### PR TITLE
Alice

### PR DESCRIPTION
**Pull request by alice-hannah**

### CODE REVIEW

File: code/02_create-scotland-file.R [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  scotland,
+  here("data", format(start_month, "%Y-%m"),
+       paste0(format(start_month, "%Y-%m"), "_scotland.rds")),
+  compress = "gz"
+)
+
+
+### 6 - Check if previous FY needs to be archived ----
+
+if(month(start_date) == 4){
```
Comment (rmccreath) [inline code/02_create-scotland-file.R]: This might need some error handling? I can't see where this comes from though.

**Review by rmccreath — APPROVED** [review]
Thanks to internet/VPN/laptop issues this was a horrendous code review. The code, however, seems perfect! I haven't been able to run everything due to the previously mentioned issues but I have checked every line and done as much execution as possible. I'm therefore happy for you to merge to the main branch. 

Feel free to request more frequent code reviews though, it's easier to manage for everyone that way :)

**Comment by alice-hannah** [comment]
Yes, apologies it was such a big one!! Thanks so much for reviewing 👍

File: code/02_create-scotland-file.R (hunk 1 above)
Comment (alice-hannah) [reply code/02_create-scotland-file.R]: Hi Russell - I've got some error handling within the `archive_data` function. Is that what you mean or something else?

Comment (rmccreath) [reply code/02_create-scotland-file.R]: I got an error when running this, I think it was to do with the variable `start_date`?

Comment (alice-hannah) [reply code/02_create-scotland-file.R]: Ah it should be `start_month`!! Silly mistake. Thanks for spotting!

Comment (rmccreath) [reply code/02_create-scotland-file.R]: No problem :) I think it was just because `start_date` had been used before but was removed.

**Pull request merged**